src/fr-webauthn/script-parser.ts

```diff
@@ -103,7 +109,7 @@ function parseWebAuthnAuthenticateText(text: string): PublicKeyCredentialRequest
       // e.g. `{ \"type\": \"public-key\",
       const type = getIndexOne(str.match(/type"{0,}:\s{0,}"([\w-]+)"/)) as 'public-key';
       // e.g. \"id\": new Int8Array([-107, 93, 68, -67, ... -19, 7, 4]).buffer
-      const idArr = ensureArray(text.match(/id"{0,}:\s{0,}new\s{0,}(Uint|Int)8Array\(([^\)]+)/));
+      const idArr = ensureArray(str.match(/id"{0,}:\s{0,}new\s{0,}(Uint|Int)8Array\(([^\)]+)/));
```

Comment:
Ah, so that was causing DSKS-572!

Reply:
Yup, on each iteration, we were accidentally grabbing the first `id` from the whole text, rather than from the individual iteration snippets.